src/2XXXR0_a_motivation_scope_and_plan_for_a_physical_quantities_and_units_library.md

```diff
@@ -559,7 +559,7 @@ and early feedback, we encourage everyone to check the documentation of the [@MP
 ## Basic Framework
 
 | Feature                                | Priority | Description                                                                                                                    |
-|----------------------------------------|:--------:|--------------------------------------------------------------------------------------------------------------------------------|
+| -------------------------------------- | :------: | ------------------------------------------------------------------------------------------------------------------------------ |
```

Comment:
Why do we need those paces here? All auto-formatting tools I know format the table in the way it was before so the next time I will save the document, those changes will be reverted anyway.

Reply:
That was done by autoformatting. I will revert it